pages/api/openPR.js

```diff
@@ -153,7 +153,7 @@ function nomineeSubmission(values, sortedSubmission) {
     SDGs: values.SDGs ? values.SDGs : [],
     sectors: values.sectors ? values.sectors : [],
     type: values.type ? values.type : [],
-    repositories: values.repositories ? [values.repositories] : [],
```

Comment:
I don't know what we were thinking here, but this is just wrong because `values.repositories` is already an array; thus this should be handled in the same way as `license`